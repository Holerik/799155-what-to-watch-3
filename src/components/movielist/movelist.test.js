// movelist.test.js
import React from 'react';
import renderer from 'react-test-renderer';
import MovieList from './movielist.jsx';

const filmsInfo = [
  {
    id: 91,
    title: `The Commuter`,
    preview: `img/commuter.jpg`,
    description: `An Insurance Salesman/Ex-Cop is caught up in a criminal conspiracy during his daily commute home`,
    genre: `Thriller`,
  }
];

it(`<MoveList /> should render small movie cadr list`, () => {
  const tree = renderer
    .create(<MovieList
      filmsInfo={filmsInfo}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
