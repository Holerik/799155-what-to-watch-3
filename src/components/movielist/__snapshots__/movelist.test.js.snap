// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MoveList /> should render small movie cadr list 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <div
      className="small-movie-card__image"
      id="91"
      onMouseOver={[Function]}
    >
      <img
        alt="An Insurance Salesman/Ex-Cop is caught up in a criminal conspiracy during his daily commute home"
        height="175"
        src="img/commuter.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        The Commuter
      </a>
    </h3>
  </article>
</div>
`;
